app/controllers/notifications_controller.rb

```diff
@@ -0,0 +1,78 @@
+class NotificationsController < ApplicationController
+  require 'json'
+  require 'twilio-ruby'
+  include Webhookable
+
+  after_filter :set_header
+
+  skip_before_action :verify_authenticity_token
+
+
+
+#does the phone number field get verified automatically (i.e. will it throw an error if a user does not enter a phone number?)
+  def plan_my_festival
+    event_list = params[:set_events]
+    event_list.each do |event_id|
+      @notification= Notification.new
+      @notification.phone_number = params[:phone_number]
+      @notification.event_id = event_id
+      @notification.save
+    end
+    redirect_to "/events/list"
+  end
+
+ def send_sms
```

Comment:
Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction)